src/components/EventDetails/EventDetails.js

```diff
@@ -174,8 +175,12 @@ const Icon = styled(CachedImage)`
   margin-bottom: ${spacing.small}px;
 `;
 
-const viewTransactionOnBlockchain = (hash: string) => {
-  Linking.openURL(TX_DETAILS_URL + hash);
+const viewTransactionOnBlockchain = (hash: string, asset?: ?string) => {
+  let url = TX_DETAILS_URL + hash;
+  if (asset && asset === 'BTC') {
```

Comment:
Can we just ..?

```suggestion
  if (asset === 'BTC') {
```

Reply:
just making sure if this function is used with/without the asset param